docs/community.md

```diff
@@ -40,7 +40,7 @@ Please make sure we are able to replicate the issue by providing the model
 definition + command + data or code + data.
 
 If your data cannot be shared, please use the `synthesize_dataset` [command line
-utility](../../user_guide/command_line_interface/#synthesize_dataset) to create
+utility](https://ludwig.ai/latest/user_guide/command_line_interface/#synthesize_dataset) to create
```

Comment:
I think we could also just do `/user_guide/command_line_interface/`; but I can't get the `code_doc_autogen` process to work locally to verify.